recipes/pyreports/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyreports" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyreports-{{ version }}.tar.gz
+  sha256: 59873c391096609c103232fe7617760810824e81c9bef366b26f1944d05f2acb
+
+build:
+  skip: true  # [win or py<37]
```

Comment:
What is the reason for skipping it on Windows? I'm just wondering if we can make it noarch.

Reply:
I don't remember why exactly but there's something in the source code that made this fail to build for win.

Reply:
done, I've reverted it